components/auction_reminders/fetch.coffee

```diff
@@ -18,6 +18,7 @@ module.exports = class AuctionReminders
     'is_auction'
     'image_versions'
     'image_url'
+    'live'
```

Comment:
Ah! I think I mentioned this in slack, but the `live` field isn't currently returned from the API at all, so we might be better off using `auction_state` here (assuming this only applies to auctions): https://github.com/artsy/gravity/blob/master/app/models/domain/sale.rb#L130

Otherwise, it will have to be added to gravity first!